Streetcode/Streetcode.WebApi/Utils/WebParsingUtils.cs

```diff
@@ -154,6 +158,8 @@ public async Task ProcessCSVfile(string extractTo, bool deleteFile = false)
 
             var remainsToParse = forParsindRows.Except(alreadyParsedRows)
                 .Select(x => x.Split(';').ToList())
+
+                // TODO take it of if you want to start global parse
                 .Take(10);
 
             var toBeDeleted = alreadyParsedRows.Except(forParsindRows);
```

Comment:
Possible multiple enumeration; Enumerate these two collections into an array or list.

Reply:
done

Reply:
The issue is still persistent.

Reply:
done